server/controllers/AutomationController.js

```diff
@@ -251,6 +253,11 @@ const updateJobResults = async (req, res) => {
     res.json({ success: true });
 };
 
+// Human test runners are able to use a checkbox to indicate no output was detected.
+// This checkbox stores 'No output was detected.' as the output value for that scenarioResult.
+const convertEmptyStringsToNoOutputMessages = outputs =>
+    outputs.map(output => (output === '' ? 'No output was detected.' : output));
```

Comment:
A little bit worried about `null` or `' '`, not enough to push for handling it now, but, just wondering what "right" answer for those two options are.

Reply:
Oh good point. I was really thinking about what the value would be with a human test run. I think we should handle null and only white space cases also. This will put the responsibility of distinguishing these cases during debugging on the sending side exclusively but it will look better from the app which seems important right now.

Reply:
Updated!